executors/sshexec/sshexec.go

```diff
@@ -56,8 +58,47 @@ var (
 	}
 )
 
+func setWithEnvVariables(config *SshConfig) {
+	var env string
+
+	// Ssh Private Key File Path
```

Comment:
I have followed the kubeexec way.

https://github.com/heketi/heketi/blob/master/executors/kubeexec/kubeexec.go#L59